modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/display/view.jsp

```diff
@@ -178,9 +178,12 @@ long formInstanceId = ddmFormDisplayContext.getFormInstanceId();
 
 						<clay:container-fluid
 							cssClass="ddm-form-builder-app ddm-form-builder-app-not-ready"
-							id='<%= ddmFormDisplayContext.getContainerId() + "container" %>'
+							id="<%= ddmFormDisplayContext.getContainerId() %>"
 						>
-							<%= ddmFormDisplayContext.getDDMFormHTML() %>
+							<react:component
+								module="admin/js/index.es"
+								props="<%= ddmFormDisplayContext.getDDMFormContext() %>"
+							/>
 
 							<aui:input name="empty" type="hidden" value="" />
 						</clay:container-fluid>
```

Comment:
@natocesarrego not having this `<aui:input name="empty" type="hidden" value="" />` might be a problem? because when React renders, it will remove all the content inside the containerId, in this case that input.

Reply:
Great warning, @matuzalemteles! I'll move the input to be below the `clay:container-fluid`. 😉